### ISSUE TITLE
Task 'WorkItem Updater' version 3 (WorkItemUpdater@3) is dependent on a Node version (10) that is end-of-life

### ISSUE THREAD

**Issue by lomeshparmar** [BlueBasher/WorkItemUpdater#159]
In Azure DevOps Pipeline Getting Warning 

##[warning]Task 'WorkItem Updater' version 3 (WorkItemUpdater@3) is dependent on a Node version (10) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance

**Comment by galadril** [comment]
Any  update on this?

**Comment by andybradycmap** [comment]
I'm also getting this error within Azure DevOps on a windows-2019 Microsoft Hosted Build Agent.

**Documentation:**
https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md

**Node Version installed on windows-2019:**
16.20.2
18.20.4
20.18.0

**Log:**
```
2024-11-14T12:36:44.4042525Z Downloading task: WorkItemUpdater (3.0.88)
2024-11-14T12:36:44.9348552Z ##[warning]Task 'WorkItem Updater' version 3 (WorkItemUpdater@3) is dependent on a Node version (10) that is end-of-life. Contact the extension owner for an updated version of the task. Task maintainers should review Node upgrade guidance: https://aka.ms/node-runner-guidance
```

**Comment by BlueBasher** [comment]
Fixed in #160
